infra/base-images/base-runner/run_fuzzer

```diff
@@ -43,11 +45,13 @@ if [[ "$FUZZING_ENGINE" = afl ]]; then
   export AFL_SKIP_CPUFREQ=1
   rm -rf /tmp/afl_output && mkdir /tmp/afl_output
   # AFL expects at least 1 file in the input dir.
-  echo input > /tmp/input/input
-  CMD_LINE="$OUT/afl-fuzz $AFL_FUZZER_ARGS -i /tmp/input -o /tmp/afl_output $@ $OUT/$FUZZER"
+  echo input > ${CORPUS_DIR}/input
+  CMD_LINE="$OUT/afl-fuzz $AFL_FUZZER_ARGS -i /tmp/
```

Comment:
Shouldn't 
-i /tmp/
be 
-i ${CORPUS_DIR}

Reply:
oops, looks like I've been typing into a wrong window and removed a pieces of the script :laughing: Thanks for catching!